.chef/knife.rb

```diff
@@ -0,0 +1,17 @@
+# See http://docs.opscode.com/config_rb_knife.html
+# for more information on knife configuration options
+
+current_dir = File.dirname(__FILE__)
+log_level :info
+log_location STDOUT
+node_name 'fakeclient'
+client_key "#{current_dir}/fakeclient.pem"
```

Comment:
Hmm, I thought it needed to exist as a file in order for Chef to not barf errors. Maybe I'm wrong though.